lib/trailblazer/operation/validate.rb

```diff
@@ -52,7 +52,7 @@ def call( ctx, ** )
 
         def validate!(options, representer:false, from: :document, params_path:nil)
           path     = "contract.#{@name}"
-          contract = options[path]
+          contract = options[path] || @constant
```

Comment:
Good work! :ok_hand: 

It should be the other way round, if `:constant` is there, it wins over the injected.

Also, what are all those Rubocop commits? And... I thought we all agreed on using double quotes everywhere for simplicity?